src/resources/js/tickets.js

```diff
@@ -116,6 +116,8 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 				$('#tribe-tickets-attendee-sortables').empty();
 				$('.tribe-tickets-attendee-saved-fields').show();
 
+				$('#ticket_bottom_right').empty();
```

Comment:
add spaces here so it's `$( '#ticket_bottom_right' )`

also, if the behavior is not negatively impacted by this, change to this:
`$( document.getElementById( 'ticket_bottom_right' ) )`